winui/AvatarView/customization.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: Customization in WinUI AvatarView control | Syncfusion
+description: Learn all about the customization features such as Height, Width, BorderBrush, Background, FontSize and CornerRadius in WinUI AvatarView control here.
```

Comment:
Description: Learn about the customization features such as Height, Width, BorderBrush, Background, FontSize, and CornerRadius in WinUI AvatarView control.